saleor/dashboard/product/views.py

```diff
@@ -475,10 +475,16 @@ def variant_details(request, product_pk, variant_pk):
     stock = variant.stock.all()
     images = variant.images.all()
     costs_data = get_variant_costs_data(variant)
+    if costs_data:
+        costs = {'min': costs_data['costs'][0],
+                 'max': costs_data['costs'][-1]}
+    else:
+        costs = {}
+
     ctx = {
         'images': images, 'product': product, 'stock': stock,
-        'variant': variant, 'costs': costs_data['costs'],
-        'margins': costs_data['margins'], 'is_empty': not stock.exists()}
+        'variant': variant, 'costs': costs, 'margins': costs_data['margins'],
```

Comment:
What if `costs_data` is empty? It seems possible given the check in line 478.

Reply:
`get_variant_costs_data()` return dict is guaranteed to always contain `costs` and `margins` keys, even if those are empty lists, so fixed condition to `if costs_data['costs']`

Reply:
Wouldn't it be more useful if it returned a named tuple or a `__slots__`-based object instead?